tools/clang-tidy-plugin/CMakeLists.txt

```
@@ -58,26 +58,6 @@ endif ()
 target_include_directories(${CataAnalyzerName} SYSTEM PRIVATE
         ${LLVM_INCLUDE_DIRS} ${CLANG_INCLUDE_DIRS})
 
-if ("${CATA_CLANG_TIDY_INCLUDE_DIR}" STREQUAL "")
-    set(CTPS_RELEASES https://github.com/jbytheway/clang-tidy-plugin-support/releases/download)
-    set(CTPS_VERSION llvm-12.0.0-r3)
-    set(CTPS_SRC ${CMAKE_CURRENT_BINARY_DIR}/clang-tidy-plugin-support)
-
-    ExternalProject_Add(clang-tidy-plugin-support
-            URL ${CTPS_RELEASES}/${CTPS_VERSION}/clang-tidy-plugin-support-${CTPS_VERSION}.tar.xz
-            URL_HASH SHA256=c84aaf35b4d5bb9130b1cc309a92146ae5c62fd69db30a6254725beac9d60990
-            SOURCE_DIR ${CTPS_SRC}
-            CONFIGURE_COMMAND ""
-            BUILD_COMMAND ""
-            INSTALL_COMMAND ""
-            TEST_COMMAND "")
-
-    add_dependencies(${CataAnalyzerName} clang-tidy-plugin-support)
-    target_include_directories(${CataAnalyzerName} SYSTEM PRIVATE ${CTPS_SRC}/include)
-else ()
-    target_include_directories(${CataAnalyzerName} SYSTEM PRIVATE ${CATA_CLANG_TIDY_INCLUDE_DIR})
-endif ()
-
 target_compile_definitions(${CataAnalyzerName} PRIVATE ${LLVM_DEFINITIONS})
```

```suggestion
separate_arguments(LLVM_DEFINITIONS_SEP UNIX_COMMAND "${LLVM_DEFINITIONS}")
target_compile_definitions(${CataAnalyzerName} PRIVATE ${LLVM_DEFINITIONS_SEP})
```

This is needed to correctly separate LLVM compile definitions on Windows. (Otherwise all definitions are passed to the compiler as a single quoted string.)